hack/run-upstream-argo-rollouts-e2e-tests.sh

```diff
@@ -46,6 +46,14 @@ if [ -z "$SKIP_RUN_STEP" ]; then
 
   set +e
 
+  # Check if the ServiceMonitor CRD exists
+  kubectl get crd/servicemonitors.monitoring.coreos.com &> /dev/null
+  retVal=$?
+  if [ $retVal -ne 0 ]; then
+      # If the CRD is not found, apply the CRD YAML
+      kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/release-0.52/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml
+  fi 
+
```

Comment:
These lines can be removed: the upstream Argo Rollouts E2E tests are not using servicemonitors.

Reply:
I removed this line from upstream argo rollouts e2e tests and the tests are failing with this error -` if kind is a CRD, it should be installed before calling Start	{"kind": "ServiceMonitor.monitoring.coreos.com", "error": "no matches for kind \"ServiceMonitor\" in version \"monitoring.coreos.com/v1\""}`